src/templates/base/2019/base.html

```diff
@@ -439,11 +465,11 @@
   {% block scripts %}
     <script nonce="{{ csp_nonce() }}">
       (function() {
-        document.querySelectorAll('.js-enable-script-remove-tag').forEach(element => {
-          var text = element.textContent;
-          var parentEl = element.parentElement;
-          parentEl.removeChild(element);
-          parentEl.innerText = text;
+        document.querySelectorAll('.js-hide').forEach(element => {
```

Comment:
Is this for the 0.00001% of users with JS disabled? Can this be solved with `<noscript>`?

Reply:
It's for those where JavaScript doesn't execute properly - either because they don't have JS enabled, or there's a syntax error, or unsupported JS syntax, or whatever. `<no script>` doesn't cover all of those. Can't remember who said it, but every browser is a no JS browser until it's downloaded parsed and executed.